README.md

```diff
@@ -8,4 +33,347 @@ UI hooks and data fetching methods built from the ground up for e-commerce appli
 - Code splitted data fetching methods for initial data population and static generation of content
 - Helpers to create the API endpoints that connect to the hooks, very well suited for Next.js applications
 
-While we're missing the docs, you can check how the package works in a live demo: [demo.vercel.store](https://demo.vercel.store/) ([repo](https://github.com/vercel/commerce))
+## Installation
+
+To install:
+
+```
+yarn add storefront-data-hooks
+```
+
+After install, the first thing you do is: <b>set your environment variables</b> in `.env.local`
+
+```sh
+BIGCOMMERCE_STOREFRONT_API_URL=<>
+BIGCOMMERCE_STOREFRONT_API_TOKEN=<>
+BIGCOMMERCE_STORE_API_URL=<>
+BIGCOMMERCE_STORE_API_TOKEN=<>
+BIGCOMMERCE_STORE_API_CLIENT_ID=<>
+```
+
+## General Usage
+
+### CommerceProvider
+
+This component is a provider pattern component creates a creates a commerce context for it's children. It takes config values for the locale and an optional `fetcherRef` object for data fetching.
```

Comment:
Typo here. "This component is a provider pattern component **creates a creates a commerce**"

Reply:
Yikes, sorry about that!